src/BaGet/Startup.cs

```diff
@@ -54,6 +55,15 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                 }
             }
 
+            if (options.StartupEnsurePackagesExistsInDatabase)
+            {
+                using (var scope = app.ApplicationServices.CreateScope())
+                {
+                    var packagesReconciler = scope.ServiceProvider.GetService<PackagesReconciler>();
+                    packagesReconciler.Reconcile().Wait(); // TODO better place for long running start routines
```

Comment:
This should be a hosted service. See [this documentation](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/hosted-services?view=aspnetcore-2.2#timed-background-tasks)